great-idea-website/css/index.css

```diff
@@ -63,4 +63,154 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+/*
+nav {
+	/*position: relative;*/
+	/*margin-left:40%;*/
+	/*display: inline-block;
+	width: 500px;*/
+	
+/*}*/
+
+/* ceneter*/
+.fullNavBar {
+	position: relative;
+	width: 1100px;
+	/*border: 2px solid #BADA55;*/
+	margin: 20px;
+}
+
+.navBar {
+	display: inline-block;
+	width: 800px;
+	padding: 10px;
+	margin: 20px;
+	/*border: 2px solid #BADA55;*/
+	/*position: absolute;*/
+}
+.navBar a {
+	margin: 35px;
+
+	color: silver;
+	text-decoration: none;
+	font-style: bold;
+}
+.navImage {
+	display: inline-block;
+	padding: 20px;
+	right: 105px;
+	position: absolute;
+	/*border: 2px solid #BADA55;*/
+}
+
+
+
+
+
+
+
+.innovation {
+	position: relative;
+	width: 1000px;
+	/*border: 2px solid #BADA55;*/
+	height: 400px;
+	top: 130;
```

Comment:
the top property would be invalid in this case, you would need to add in px ( 130px)